src/frida/repl.py

```diff
@@ -27,15 +27,20 @@ def __init__(self):
             self._history = FileHistory(os.path.join(os.path.expanduser('~'), '.frida_history'))
             self._completer = FridaCompleter(self)
             self._cli = None
+            self._last_change_id = 0
+            self._script_monitor = None
 
             super(REPLApplication, self).__init__(self._process_input, self._on_stop)
 
         def _add_options(self, parser):
             parser.add_option("-l", "--load", help="load SCRIPT", metavar="SCRIPT",
                 type='string', action='store', dest="user_script", default=None)
+            parser.add_option("-w", "--watch", help="watch SCRIPT for changes and automatically reload it",
+                action='store_true', dest="watch", default=False)
 
         def _initialize(self, parser, options, args):
-            self._user_script = options.user_script
+            self._user_script = os.path.abspath(options.user_script)
```

Comment:
Won't this crash if the user doesn't pass `-l` and `options.user_script` is `None`?